browser/ui/webui/skus_internals_ui.cc

```diff
@@ -94,25 +95,36 @@ void SkusInternalsUI::GetVpnState(GetVpnStateCallback callback) {
 #if !BUILDFLAG(IS_ANDROID)
   dict.Set("Last connection error", GetLastVPNConnectionError());
 #endif
-  dict.Set("Order", GetVPNOrderInfo());
+  auto* profile = Profile::FromWebUI(web_ui());
+  if (!brave_vpn::IsBraveVPNEnabled(profile->GetPrefs())) {
+    base::Value::Dict empty;
+    dict.Set("Order", empty);
```

Comment:
Dict should be rvalue.
```suggestion
    dict.Set("Order", base::Value::Dict());
```